modules/zip/test/zip-utils/local-file-header.spec.js

```diff
@@ -1,7 +1,7 @@
 import test from 'tape-promise/tape';
 import {DATA_ARRAY} from '@loaders.gl/i3s/test/data/test.zip';
 import {parseZipLocalFileHeader} from '../../src/parse-zip/local-file-header';
-import {DataViewFile} from '../../src/parse-zip/data-view-file';
+import {DataViewFile} from '../../src/classes/data-view-file';
```

Comment:
Nit: `classes` as directory name doesn't provide much information, seems like a missed opportunity to make the code more self documenting?
```suggestion
import {DataViewFile} from '../../src/file-provider/data-view-file';
```